<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Customer</class>
 <widget class="QWidget" name="Customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnMenu">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(135, 125, 120);
        color: #fff;
        font-size: 13px;
        font-weight: bold;
        border-top-right-radius: 15px;
        border-top-left-radius: 0px;
        border-bottom-right-radius: 0px;
        border-bottom-left-radius: 15px;
        padding: 10px;

}</string>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMyOrder">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(135, 125, 120);
        color: #fff;
        font-size: 13px;
        font-weight: bold;
        border-top-right-radius: 15px;
        border-top-left-radius: 0px;
        border-bottom-right-radius: 0px;
        border-bottom-left-radius: 15px;
        padding: 10px;

}</string>
   </property>
   <property name="text">
    <string>My Order</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>631</width>
     <height>521</height>
    </rect>
   </property>
   <widget class="QWidget" name="page_0_menu">
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>601</width>
       <height>480</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>599</width>
        <height>485</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="menuGrid"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnPlaceOrder">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Order and Pay</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 14pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_1_place_order">
    <widget class="QTableWidget" name="tableWidget_selectedItems">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>401</width>
       <height>201</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Item</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Price</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantity</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnPayAndOrder">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>340</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pay to place your order</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>340</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>Total:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTotalPrice">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2_my_order">
    <widget class="QTableView" name="table_my_order">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>601</width>
       <height>361</height>
      </rect>
     </property>
     <property name="autoScrollMargin">
      <number>11</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 14pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>My ordering</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3_order_details">
    <widget class="QLabel" name="label_details_code">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>551</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_order_details">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>561</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="labelAccount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
